src/core/core.controller.js

```diff
@@ -147,8 +147,8 @@ function updateConfig(chart) {
 	chart.tooltip.initialize();
 }
 
-function positionIsHorizontal(position) {
-	return position === 'top' || position === 'bottom';
+function positionIsHorizontal(position, axis = undefined) {
```

Comment:
no need to specify `undefined` as default, it will be undefined if not provided.